diff --git a/node_modules/expo-updates/scripts/create-manifest-android.gradle b/node_modules/expo-updates/scripts/create-manifest-android.gradle
index 4a364e1..0c8f26a 100644
--- a/node_modules/expo-updates/scripts/create-manifest-android.gradle
+++ b/node_modules/expo-updates/scripts/create-manifest-android.gradle
@@ -10,7 +10,7 @@ void runBefore(String dependentTaskName, Task task) {
   }
 }

-def expoUpdatesDir = ["node", "-e", "console.log(require('path').dirname(require.resolve('expo-updates/package.json')));"].execute([], projectDir).text.trim()
+def expoUpdatesDir = ["node", "-e", "console.log(require('path').dirname(require.resolve('expo-updates/package.json')));"].execute(null, projectDir).text.trim()

 def config = project.hasProperty("react") ? project.react : [];
 def nodeExecutableAndArgs = config.nodeExecutableAndArgs ?: ["node"]
